host/CMakeLists.txt

```diff
@@ -488,3 +488,321 @@ install(TARGETS oehostapp EXPORT openenclave-targets)
+    crypto/bcrypt/hmac.c
+    crypto/bcrypt/key.c
+    crypto/bcrypt/pem.c
+    crypto/bcrypt/random.c
+    crypto/bcrypt/rsa.c
+    crypto/bcrypt/sha.c
+    windows/hostthread.c
+    windows/syscall.c
+    windows/time.c)
+else()
+  message(FATAL_ERROR "Unknown OS. Only supported OSes are Linux and Windows")
+endif()
+
+if (OE_SGX AND WIN32)
+  # Use clang to compile the oe_enter function that calls ENCLU.
+  # Windows debuggers (WinDbg and Visual Studio Debugger) require that the rbp
+  # is set up like a Linux x64 ABI style frame pointer by the oe_enter function.
+  # Since oe_enter function is setting up the frame-pointer, enter.c must be
+  # compiled with the -fomit-frame-pointer flag so that the compiler does not
+  # set up the frame pointer.
+  add_custom_command(
+     OUTPUT enter.obj
+     DEPENDS sgx/enter.c
+     COMMAND clang  -c -O2 -fomit-frame-pointer -m64
+             -I${PROJECT_SOURCE_DIR}/include
+             ${CMAKE_CURRENT_SOURCE_DIR}/sgx/enter.c
+             -o enter.obj)
+endif()
+
+# SGX specific files.
+if (OE_SGX)
+	list(APPEND PLATFORM_OEMR_SRC
+		#    ../common/sgx/endorsements.c
+		#    ../common/sgx/qeidentity.c
+		#    ../common/sgx/quote.c
+		#    ../common/sgx/report.c
+		#    ../common/sgx/revocation.c
+		#    ../common/sgx/sgxcertextensions.c
+		#    ../common/sgx/tcbinfo.c
+		#    ../common/sgx/tlsverifier.c
+		#    ../common/sgx/verifier.c
+		#    sgx/hostverify_report.c
+		#    sgx/sgxquoteprovider.c
+
+		#sgx/calls.c
+    sgx/create.c
+	    sgx/elf.c
+	#    sgx/enclave.c
+	#    sgx/enclavemanager.c
+	#    sgx/exception.c
+	sgx/load.c
+	sgx/loadelf.c
+    sgx/loadpe.c
+	#    sgx/ocalls.c
+	#	sgx/quote.c
+	#    sgx/registers.c
+	#    sgx/report.c
+	#    sgx/sgx_u_wrapper.c
+    sgx/sgxload.c
+    sgx/sgxmeasure.c
+	#    sgx/sgxquote.c
+	sgx/sgxsign.c
+	sgx/sgxtypes.c
+	#    sgx/switchless.c
+	#    sgx/switchless_u_wrapper.c)
+	)
+
+
+  # OS specific as well.
+  if (UNIX)
+	  list(APPEND PLATFORM_OEMR_SRC
+		  #        sgx/linux/sgxquoteproviderloader.c
+
+		  #      sgx/linux/aep.S
+		  #      sgx/enter.c
+		  #      sgx/linux/exception.c
+		  #      sgx/linux/sgxioctl.c
+		  #      sgx/linux/switchless.c
+		sgx/linux/xstate.c)
+  else()
+	  list(APPEND PLATFORM_OEMR_SRC
+		  #      sgx/windows/sgxquoteproviderloader.c
+		  #
+		  #      sgx/windows/aep.asm
+		  #      ${CMAKE_CURRENT_BINARY_DIR}/enter.obj
+		  #      sgx/windows/exception.c
+		  #      sgx/windows/switchless.c
+      sgx/windows/xstate.c)
+  endif()
+
+  set(PLATFORM_FLAGS "-m64")
+elseif(OE_TRUSTZONE)
+	list(APPEND PLATFORM_OEMR_SRC
+    optee/log.c)
+
+  if (UNIX)
+	  list(APPEND PLATFORM_OEMR_SRC
+      optee/linux/enclave.c)
+  else()
+    message(FATAL_ERROR "OP-TEE is not yet supported on platforms other than Linux.")
+  endif()
+
+  set(PLATFORM_FLAGS "")
+endif()
+
+if (OE_SGX AND WIN32)
+  # oedebugrt is accessed via a bridge on Win32 and need not be linked.
+  list(APPEND PLATFORM_OEMR_SRC
+    sgx/windows/debugrtbridge.c)
```

Comment:
This seems off. oesign is not concerned with debugging and would not need to do include debugrt stuff.

Reply:
updated, thanks.